docs/topics/mta-6-installing-web-console-on-openshift.adoc

```diff
@@ -68,7 +68,58 @@ You can install the {ProductName} ({ProductShortName}) and the {WebName} on Open
 . Click *Operators → Installed Operators* to verify that the {ProductShortName} Operator appears in the `openshift-mta` project with the status `Succeeded`.
 . Click the *{ProductShortName}* Operator.
 . Under *Provided APIs*, locate *Tackle*, and click *Create Instance*.
-. Review options - the default selections should be OK, but be sure to validate the system requirements for storage, memory, and cores - and then click *Create*.
++
+The *Create Tackle* window opens in *Form view*.
+. Review the CR settings. The default choices should be acceptable, but make sure to check the system requirements for storage, memory, and cores.
+. If you prefer to work directly with the YAML file, click *YAML view* and review the CR settings that are listed in the `spec` section of the YAML file.
++
+The most commonly used CR settings are listed in this table:
++
+.Tackle CR settings
+[cols="40%,15%,55%", options="header"]
+|====
+|Name
+|Default
+|Description
+
+|`feature_auth_required`
+|True
+|Whether or not keycloak authorization is required (single user/"`noauth`")
+
+|`feature_isolate_namespace`
+|True
+|Whether or not namespace isolation using network policies is enabled
```

Comment:
@RichardHoch 
Same as above. I would suggest "Flag to indicate whether namespace isolation using..."

Reply:
@sbeskin-redhat -- same as above.

Reply:
@RichardHoch 
Same as above